src/components/TransactionFailText/index.tsx

```diff
@@ -52,8 +54,10 @@ export const TransactionFailText = ({
       <Paragraph color="error" className={classes.executionWarningRow}>
         <Img alt="Info Tooltip" height={16} src={InfoIcon} className={classes.warningIcon} />
 
-        {isOwner ? (
+        {isGranted ? (
           <>This transaction will most likely fail. {errorMessage}</>
+        ) : isWrongChain ? (
+          <>Your wallet is connected to a wrong chain.</>
```

Comment:
"Your wallet is connected to the wrong chain."

Reply:
Fixed, thanks!